src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/ClientConfigurationSettings.java

```diff
@@ -103,7 +103,7 @@ public static class ClientSideOnTypeFormattingSettings {
          * The specific close brace characters that should trigger client-side on-type formatting. Defaults to the
          * language's close brace characters.
          */
-        public String formatOnCloseBraceCharacters = null;
+        public String formatOnCloseBraceCharacters = "";
```

Comment:
Changed these to default to the empty string instead of null. Downstream consumers are just checking for an empty or null string, so it's a no-op other than that we don't have to check for null when collecting these from language server definitions before appending.